qiskit/transpiler/passes/basis/unroller.py

```diff
@@ -68,6 +70,22 @@ def run(self, dag):
                 else:
                     continue
 
+            if isinstance(node.op, ControlFlowOp):
+                new_params = []
+                for param in node.op.params:
+                    if isinstance(param, QuantumCircuit):
+                        # TODO: check whether unrolling is necessary
```

Comment:
I wouldn't worry too much about the comment - the idea of the unroller is already quite inefficient anyway, and each op should only be accessed once, so the additional cost probably isn't too high.